tests/TestCase/Auth/Storage/SessionStorageTest.php

```diff
@@ -31,7 +32,7 @@ class SessionStorageTest extends TestCase
      *
      * @return void
      */
-    public function setUp()
+    public function setUp(): void
```

Comment:
Do these help at all? I don't see return hints helping much on test methods.

Reply:
They also dont hurt.

Reply:
Wasted effort does hurt :)

Reply:
Thats why you use automated sniffers. Always.

Reply:
They were added by an automated sniff :) I just wanted to keep it consistent

Reply:
Exactly, consistency across the code base is key.

Reply:
If they can be autofixed then ok. I know I will forget to add these all the time :blush: